src/JMS/Serializer/Context.php

```diff
@@ -106,6 +108,37 @@ public function getExclusionStrategy()
         return $this->exclusionStrategy;
     }
 
+    /**
+     * @param string $class
+     * @return bool
+     */
+    public function hasExclusionStrategy($class)
+    {
+        if (isset($this->hasExclusionStrategy[$class])) {
+            return $class;
+        }
+        $strategies = array();
+        if ($this->exclusionStrategy) {
+            $strategies[] = $this->exclusionStrategy;
+        }
+
+        while (count($strategies)>0) {
+            $strategy = array_shift($strategies);
+
+            if ($strategy instanceof $class) {
+                return $this->hasExclusionStrategy[$class] = true;
+            } elseif ($strategy instanceof DisjunctExclusionStrategy) {
+                $ref = new \ReflectionProperty($strategy, 'delegates');
```

Comment:
i prefer to add just the method... `DelegatingStrategyInterface` looks an overkill to me